spec/javascript/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -20,14 +20,19 @@ const ACUANT_CAPTURE_SUCCESS_RESULT = {
     width: 1748,
     height: 1104,
   },
-  cardType: 1,
+  cardtype: 1,
   dpi: 519,
   moire: 99,
   moireraw: 99,
   glare: 100,
   sharpness: 100,
 };
 
+const ACUANT_CAPTURE_CARDTYPE_ERROR = { ...ACUANT_CAPTURE_SUCCESS_RESULT };
+
+delete ACUANT_CAPTURE_CARDTYPE_ERROR.cardtype;
+ACUANT_CAPTURE_CARDTYPE_ERROR.cardType = 2;
```

Comment:
I'm not entirely clear why we're setting the casing here this way, since the code never references it. To exercise the new `default` case, would we want to use the correct casing on the property, but a value that's not accounted for (e.g. `3`, `4`, or `undefined` explicitly?).

```suggestion
const ACUANT_CAPTURE_CARDTYPE_ERROR = { ...ACUANT_CAPTURE_SUCCESS_RESULT, cardtype: undefined };
```